feature/interface/ip/ipv6_slaac_link_local/README.md

```diff
@@ -0,0 +1,29 @@
+# RT-5.9: IPv6 Link Local generated by SLAAC
+
+## Summary
+
+Enable IPv6 on interface level so ipv6 address of link-local scope is generated/assigned by SLAAC. Verify the link local IPv6 address generated, exists by checking the Openconfig path
+
+## Testbed type
+
+*   https://github.com/openconfig/featureprofiles/blob/main/topologies/dut.testbed
```

Comment:
I believe this should be https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed

TESTBED_DUT has 0 links:
https://github.com/openconfig/featureprofiles/blob/1139171167f376654217968747d632f1baae2f3b/topologies/dut.testbed#L4